tests/com/twitter/intellij/pants/components/impl/PantsProjectCacheTest.java

```diff
@@ -43,7 +43,15 @@ public void run() {
   @Override
   protected void tearDown() throws Exception {
     PantsProjectCacheImpl.getInstance(getProject()).projectClosed();
-    super.tearDown();
+    try {
+      super.tearDown();
+    }
+    catch (Exception e) {
+      // TODO: something in the test framework isn't disposed.
```

Comment:
It looks like this is `PsiTestUtil.addSourceRoot(getModule(), VfsUtil.createDirectoryIfMissing(root, "bar"));`. This directory is likely being created and thus needs to be destroyed as well. Eating these exceptions will probably mask actual resource leaks.

Reply:
You are right, although I haven't figured out how to do that yet.

Since this isn't critical since it's an issue from test framework, I've decided to unblock people from getting updates for now.

Reply:
Separate PR up: https://github.com/pantsbuild/intellij-pants-plugin/pull/412